test/unit/helpers/z_schema-express.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+
+var z_schema_express = require('../../../helpers/z_schema-express');
+
+describe('z_schema.express', function () {
+
+	it('returns true because the schema is valid', function () {
+		var z_schema = {
+			'validate' : function (value, schema, cb) {
```

Comment:
you could use `validate: sinon.stub().callsArgWith(2, null, true)`

Reply:
M. Please excuse my ignorance on this... I'm far from a JS expert... I am hoping that you can help me understand.

In the validate method I need to call the cb(), as its what is set in the impl class, in order to fire off the sanitize function. It is then below in which I validate the response.

So... should it be in the sanitize method I leverage sinon.stub()...?

Thanks in advance.
R